Using-Mastodon/FAQ.md

```diff
@@ -64,75 +71,147 @@ Try [TootyFruity](https://play.google.com/store/apps/details?id=ch.kevinegli.too
+
 #### What’s trending?
-Follow the esteemed [@TrendingBot@mastodon.social](https://mastodon.social/@TrendingBot).
+To find out what’s trending, follow the esteemed [@TrendingBot@mastodon.social](https://mastodon.social/@TrendingBot).
 
 #### What’s “CW”?
 “CW” stands for “Content Warning.” You can use it to hide your toot, like a spoiler.
 
+#### How does the “Mark Image as Sensitive” feature work (“NSFW”)?
+When you add a picture to your post using the camera icon, an additional toggle will appear [typically as "NSFW"]. Clicking this will hide your image behind a "Sensitive Content" warning, preventing others from seeing your images until they click on them.
+
 #### What’s with the pineapples?
 They’re tasty, and they make people smile. Just go with it.
 
+#### I see “Awoo” a lot, what does that mean?
+Try saying it out loud. It’s fun!
+
 #### How do I delete my account?
 At the moment, you will have to toot the admin of your instance for help on that one.
 
 #### Who is my admin, and how do I contact/follow them?
 Click on **Getting Started**, then **Extended Information**. An info page displays. Hopefully, they put their contact information in there!
 
-![Getting Started](https://octodon.social/media/TKlZpSNRRcHBOEFNlGs) ![Extended Information](https://octodon.social/media/D7bfl1NYzNy4XIT8nLs) ![Info Page](https://octodon.social/media/zvoI8yUt4uurZcKqFRk)
+<img src="screenshots/toolbar-getting_started.png" alt="Getting Started Button" height="200"/>
+<img src="screenshots/getting_started-extended_information.png" alt="Extended Information" height="200"/>
+<img src="screenshots/admin_info.png" alt="Admin Info" height="200"/>
 
 #### Wow, I keep seeing offensive stuff from a particular instance, is there a way to block *all* of it?
 Not without a lot of clicking, sorry. But, your instance admin can do it! Please contact her.
 
+#### I blocked someone, why can I still see their posts?
+Blocking doesn’t catch when toots are boosted.
+
+#### Is Mastodon moderated?
+Each instance handles moderation differently, and each has their own moderators. It’s okay to ask what the rules are for the instance you are on. Usually the rules are posted on the instance's about page ([like the “more” page at Mastodon.social](https://mastodon.social/about/more))
+
+#### How do I report offensive content?
+Beneath each post, you will see three dots. Clicking on those will allow you to expand a post or report the content. When reporting content, select all posts that need moderator attention.
+
+#### How do I handle harassment?
+Harassment is taken seriously by the admins and moderators of instances and should be reported through the post-reporting system which can be found via the three dots under every post or writing your admin/mod directly.
```

Comment:
"If your admin takes harassment seriously, you can report it to them through the post-reporting system. This can be found through the ellipsis (“...”) beneath the post, or by contacting your admin directly."